igneous/tasks/skeleton.py

```diff
@@ -989,14 +1248,15 @@ def process_skeletons(self, unfused_skeletons, in_place=False):
         if attr['data_type'] == 'float32' 
       ]
       skel = skel.consolidate()
-      if self.max_cable_length is not None and skel.cable_length() > self.max_cable_length:
-        skeletons[label] = skel.to_precomputed()
-      else:
-        skeletons[label] = kimimaro.postprocess(
-          skel, 
-          dust_threshold=self.dust_threshold, # voxels 
-          tick_threshold=self.tick_threshold, # nm
-        ).to_precomputed()
+      # if self.max_cable_length is not None and skel.cable_length() > self.max_cable_length:
```

Comment:
Maybe add a parameter for postprocessing type or "skip_postprocess"? I think the main problem here was loop elimination is O(V^2)